common/djangoapps/entitlements/models.py

```diff
@@ -452,3 +452,16 @@ def __unicode__(self):
             self.support_user,
             self.reason,
         )
+
+    @classmethod
+    def get_support_reasons_list(cls):
+        """
+        Method for retrieving a serializable version of the entitlement support reasons
+
+        Returns Array of reasons
+        """
+        return [
+          {'value': value, 'label': label} 
```

Comment:
Complete side note but in ES6 (JavaScript) you can do something like

```javascript
const value = 'value';
const label = 'label';

const combined = {
  value,
  label,
};

// combined = { value: 'value', label: 'label' }
```

![image](https://user-images.githubusercontent.com/8136030/36922687-c702a688-1e36-11e8-9206-ccce8e9ff5ed.png)
- [RunKit](https://runkit.com/jaebradley/5a99c1e574961c0012630c16)

and I kinda wish you could do the same in Python (and maybe you can, and my eugooglies haven't been up to scratch) like

```python
return [ { value, label } for value, label in cls.ENTITLEMENT_SUPPORT_REASONS ]
```

Reply:
I don't think python supports that kind of implicit naming. pretty sure thats an es6 sugar